CHANGELOG.md

```diff
@@ -28,7 +38,7 @@ All notable changes to `src-cli` are documented in this file.
 
 - Fix network timeout in `src users clean` occuring in instances with many users  [#901](https://github.com/sourcegraph/src-cli/pull/901)
 - Aligned parsing of spec file parameter of `src batch repos` with other commands. [#919](https://github.com/sourcegraph/src-cli/pull/919)
-- batches: Remove empty log outputs. [#923](https://github.com/sourcegraph/src-cli/pull/923)
```

Comment:
Could we maybe say where we're removing the empty log outputs from in this entry? Like "Remove empty log outputs during batch spec execution" (if that's accurate)?

Reply:
Sure. I'll fix that. Thanks